File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/ride/StartRideDataResponse.java
package com.lattis.ellipse.data.network.model.response.ride;

import com.google.gson.annotations.SerializedName;

public class StartRideDataResponse {

    @SerializedName("trip_id")
    private int tripId;

    @SerializedName("do_not_track_trip")
    private Boolean do_not_track_trip;

    public int getTripId() {
        return tripId;
    }

    public Boolean getDo_not_track_trip() {
        return do_not_track_trip;
    }

    public void setTripId(int tripId) {
        this.tripId = tripId;
    }


}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/ride/StartRideDataResponse.kt
package com.lattis.data.entity.response.ride

import com.google.gson.annotations.SerializedName

class StartRideDataResponse {
    @SerializedName("trip_id")
    var tripId = 0

    @SerializedName("do_not_track_trip")
    val do_not_track_trip: Boolean? = null

}